slackminion/bot.py

```diff
@@ -342,21 +341,22 @@ async def _event_message(self, **payload):
         if not msg:
             return
 
+        self.log.info(payload)
```

Comment:
Log is as INFO only if it's a relevant message.

Reply:
> discussed offline, and the are actually debug logs so they should be left as debug and instead control the log levels being recorded via the standard python tooling around this.

I think it's fine leaving as debug, however I would appreciate this to be included (even as debug) in the `dispatcher.push` function (since it's where commands are identified apart the rest of regular Slack messages).